server/src/room/id.rs

```diff
@@ -0,0 +1,117 @@
+use once_cell::sync::Lazy;
+use std::error::Error;
+use std::fmt::{self, Write};
+use std::str::FromStr;
+
+// Alphanumerics, excluding iIoO01, since they look too similar.
```

Comment:
Can you exclude `l` too? That's not distinguishable from `I` in most sans-serif fonts.